portal-impl/src/portal.properties

```diff
@@ -6993,13 +6993,21 @@
     #
     search.container.page.iterator.max.pages=25
 
+    #
+    # These properties are deprecated and will be removed in a future release,
+    # they only affect the small number of non-lexicon searches such as Define
```

Comment:
I'm not sure about the term "non-lexicon" since it is not clear what that means in this context. Perhaps we could say explicitly "searches still using legacy markup"?